src/create-support-ticket.ts

```diff
@@ -107,98 +116,130 @@ export const sendSqsMessage = async (
+      !ZENDESK_GROUP_ID_KEY ||
+      !ZENDESK_TICKET_FORM_ID ||
+      !ACTIVITY_LOG_TABLE
+    ) {
+      throw new Error(
+        "Not all environment variables required to successfully send to Zendesk are provided."
+      );
+    }
+
+    validateSuspiciousActivity(input);
+    const zendeskUserName = await getSecret(ZENDESK_API_USER_KEY, {
+      maxAge: 900,
+    });
+    const zendeskUserPassword = await getSecret(ZENDESK_API_TOKEN_KEY, {
+      maxAge: 900,
+    });
+    const zendeskGroupId = await getSecret(ZENDESK_GROUP_ID_KEY, {
+      maxAge: 900,
+    });
+    const zendeskURL = await getSecret(ZENDESK_API_URL_KEY, {
+      maxAge: 900,
+    });
+    const zendeskTicketFormId = await getSecret(ZENDESK_TICKET_FORM_ID, {
+      maxAge: 900,
+    });
+    const zendeskTicketTags = ZENDESK_TAGS_KEY
+      ? await getSecret(ZENDESK_TAGS_KEY, {
           maxAge: 900,
-        });
-        const zendeskTicketTags = ZENDESK_TAGS_KEY
-          ? await getSecret(ZENDESK_TAGS_KEY, {
-              maxAge: 900,
-            })
-          : undefined;
-
-        if (
-          !zendeskUserName ||
-          !zendeskUserPassword ||
-          !zendeskGroupId ||
-          !zendeskURL
-        ) {
-          throw new Error("Required zendesk secrets not configured");
-        }
-
-        const tags = (zendeskTicketTags as string)?.split(",");
-        const ticket: CreateTicketPayload = {
-          ticket: {
-            subject: "REPORT_SUSPICIOUS_ACTIVITY -OLH TEST IGNORE",
-            comment: {
-              html_body: formatCommentBody(receivedEvent),
-            },
-            group_id: Number(zendeskGroupId),
-            tags,
-          },
-        };
-
-        await createTicket(
-          ticket,
-          zendeskURL as string,
-          zendeskUserName as string,
-          zendeskUserPassword as string
-        );
-      } catch (error: unknown) {
-        console.error(
-          `[Error occurred], unable to send suspicious activity event with ID: ${eventIdentifier} to Zendesk, ${
-            (error as Error).message
-          }`
-        );
-        const { AWS_REGION } = process.env;
-        const client = new SQSClient({ region: AWS_REGION });
-        const message: SendMessageRequest = {
-          QueueUrl: DLQ_URL,
-          MessageBody: record.Sns.Message,
-        };
-        const result = await client.send(new SendMessageCommand(message));
-        console.error(
-          `[Message sent to DLQ] with message id = ${result.MessageId}`,
-          error
+        })
+      : undefined;
+
+    if (
+      !zendeskUserName ||
+      !zendeskUserPassword ||
+      !zendeskGroupId ||
+      !zendeskURL ||
+      !zendeskTicketFormId
+    ) {
+      throw new Error("Required zendesk secrets not configured");
+    }
+
+    const tags = (zendeskTicketTags as string)?.split(",");
+    const ticket: CreateTicketPayload = {
+      ticket: {
+        subject: "REPORT_SUSPICIOUS_ACTIVITY -OLH TEST IGNORE",
```

Comment:
is there a ticket somewhere to change this once we go live?

Reply:
Good spot, the test ignore has now been removed from the subject title, Because we are sending to sandbox and it is not going on live support team queue.